src/ukify/ukify.py

```diff
@@ -1275,6 +1277,16 @@ def config_example(self) -> tuple[Optional[str], Optional[str], Optional[str]]:
         config_key = 'UKI/SBAT',
     ),
 
+    ConfigItem(
```

Comment:
This commit (abffcc685ecd5098ea95b4793d946514ba928bb4) duplicates the `--sbat` option, making the CI sad. This is because our tree already contains a8b645dec8e6abf4c9ba0c93a6a0088953a2155e, which includes bcc54c0a5743a309654823eeb72b5b5dc24c9062, that builds on this commit, so this commit can be safely dropped.